build_forklift_bazel.sh

```diff
@@ -68,12 +68,19 @@ if [ "${PROVIDER_NAME}" = "vsphere" ]; then
         --action_env VIRT_V2V_DONT_REQUEST_KVM=true "
 fi
 
+if [ "${PROVIDER_NAME}" = "ova" ]; then
+    ACTION_ENV="$ACTION_ENV --action_env VIRT_V2V_IMAGE=quay.io/kubev2v/forklift-virt-v2v-stub:${REGISTRY_TAG} \
```

Comment:
yeah the real virt-v2v should be used, changed for ova, the issue will be with the `all` target, since the vmware using the stab we need to figure how to include the real one as well.

Reply:
ack